core/src/components/popover/popover.tsx

```diff
@@ -457,19 +457,24 @@ export class Popover implements ComponentInterface, PopoverInterface {
     };
 
     const { inline, delegate } = this.getDelegate(true);
+    const lazyBuild = hasLazyBuild(el);
     this.usersElement = await attachComponent(delegate, el, this.component, ['popover-viewport'], data, inline);
-    hasLazyBuild(el) && (await deepReady(this.usersElement));
+    lazyBuild && (await deepReady(this.usersElement));
 
     if (!this.keyboardEvents) {
       this.configureKeyboardInteraction();
     }
     this.configureDismissInteraction();
 
-    // TODO: FW-2773: Apply this to only the lazy build.
-    /**
-     * ionMount only needs to be emitted if the popover is inline.
-     */
-    this.ionMount.emit();
+    if (lazyBuild) {
```

Comment:
The tech debt ticket also mentions wrapping the `await waitOneFrame` (below the `ionMount` emit) in this check; was that skipped intentionally?